src/main/kotlin/com/hedvig/underwriter/serviceIntegration/productPricing/dtos/QuoteDto.kt

```diff
@@ -55,7 +55,7 @@ data class QuoteDto(
                 id = quote.id,
                 createdAt = quote.createdAt,
                 price = quote.price,
-                currency = quote.currency,
+                currency = quote.currencyWithFallbackOnMarket,
```

Comment:
This feels 1% more correct 
```suggestion
                currency = quote.currency ?: quote.currencyWithFallbackOnMarket,
```

Reply:
This is made implicit in the `quote.currencyWithFallbackOnMarket` getter...

Reply:
Ah then it's perfect